src/main/java/duke/entity/command/DoneCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.entity.command;
+
+import duke.entity.task.Task;
+import duke.entity.TaskList;
+import duke.handler.Storage;
+import duke.handler.Ui;
+
+import java.io.IOException;
+import java.util.List;
+
+public class DoneCommand extends Command {
```

Comment:
I like how you've chosen to model the commands as `classes`. I did not think of it this way and by reviewing your PR, I was able to learn about this additional way of modelling the duke application. Thanks!